TSPL.docc/LanguageGuide/ControlFlow.md

```diff
@@ -844,6 +964,40 @@ this `switch` statement uses a `default` case
 to match all characters other than `a` and `z`.
 This provision ensures that the `switch` statement is exhaustive.
 
+Like `if` statements,
+`switch` statements also have an expression form:
+
+```swift
+let anotherCharacter: Character = "a"
+let message = switch anotherCharacter {
+case "a":
+    "The first letter of the alphabet"
```

Comment:
We should be specific about the alphabet type.

Reply:
Fixed throughout the chapter in 0d492dc5.

I believe the period was intentionally omitted in the existing example, which I matched, because the text is a noun fragment.  Looking at the output of `git grep 'print("'` that's how text is printed throughout the book.  If that's something we want to change now, let's track that as its own corpus-wide rewrite.